src/javascript/binary/websocket_pages/user/new_account/real_acc_opening.js

```diff
@@ -16,14 +16,22 @@ const RealAccOpening = (() => {
             if (AccountOpening.redirectAccount(account_type_ico)) return;
 
             BinarySocket.wait('landing_company').then(() => {
-                const form_id = '#frm_real';
+                const form_id   = '#frm_real';
+                let validations = AccountOpening.commonValidations()
+                    .concat(AccountOpening.selectCheckboxValidation(form_id));
+
+                // If user is allowed to open ICO account give an option to choose account type.
+                if(Client.canOpenICO() && !account_type_ico && !Client.hasAccountType('ico')) {
+                    const more_validations = AccountOpening.accountTypeInit(form_id);
```

Comment:
since it's only used here there's no need for accountTypeInit to return the validations, we can just add the validations here